domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -280,17 +276,30 @@ def on_get_ep400_contentions(self, event, pending_contentions_response=None):
 
     @running_check_pending_is_open.enter
     def on_check_pending_claim_is_open(self, event, pending_contentions_response, ep400_contentions_response):
-        request = get_claim.Request(claim_id=self.job.pending_claim_id)
+        self.__get_open_claim(self.job.pending_claim_id, 'Pending')
+        self.send(event=event, pending_contentions_response=pending_contentions_response, ep400_contentions_response=ep400_contentions_response)
+
+    @running_check_ep400_is_open.enter
+    def on_check_ep400_claim_is_open(self, event, pending_contentions_response, ep400_contentions_response):
+        self.__get_open_claim(self.job.ep400_claim_id, 'EP400')
+        self.send(event=event, pending_contentions_response=pending_contentions_response, ep400_contentions_response=ep400_contentions_response)
+
+    def __get_open_claim(self, claim_id: int, claim_type: str) -> ClaimDetail | None:
```

Comment:
I'm not the biggest fan of `__private` methods unless it's guarding against a real risk of outside code dangerously mucking with internal state. What do you think?

Reply:
Yeah i suppose methods are never truly private anyhow, and in this case the method is not changing state, so I'm cool with changing it. Encapsulation isn't strictly enforced in python and I haven't been marking all the others as private up to this point either 😅